### PR TITLE
Add Hyper-V Hypervisor Logical Processor metrics

### PR DESCRIPTION
**Pull request by gloyka**
Add Win32_PerfRawData_HvStats_HyperVHypervisorLogicalProcessor WMI class to monitor Hyper-V Hypervisor Logical Processor. The class shows metrics for actual CPU utilization on hypervisor hosts.

This addresses https://github.com/prometheus-community/windows_exporter/issues/444

Example output:
```
# HELP windows_hyperv_host_lp_guest_run_time The percentage of time spent by the processor in guest code
# TYPE windows_hyperv_host_lp_guest_run_time gauge
windows_hyperv_host_lp_guest_run_time{core="0"} 4.288535665036e+12
windows_hyperv_host_lp_guest_run_time{core="1"} 3.8546816773e+11
# HELP windows_hyperv_host_lp_hypervisor_run_time The percentage of time spent by the processor in hypervisor code
# TYPE windows_hyperv_host_lp_hypervisor_run_time gauge
windows_hyperv_host_lp_hypervisor_run_time{core="0"} 9.95114252719e+11
windows_hyperv_host_lp_hypervisor_run_time{core="1"} 2.50063864275e+11
# HELP windows_hyperv_host_lp_total_run_time The percentage of time spent by the processor in guest and hypervisor code
# TYPE windows_hyperv_host_lp_total_run_time gauge
windows_hyperv_host_lp_total_run_time{core="0"} 5.283649917755e+12
windows_hyperv_host_lp_total_run_time{core="1"} 6.35532032005e+11
```

### CODE REVIEW

File: collector/hyperv.go [hunk 1]
```diff
@@ -309,6 +314,27 @@ func NewHyperVCollector() (Collector, error) {
 
 		//
 
+		HostLPGuestRunTime: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, buildSubsystemName("host_lp"), "guest_run_time"),
```
Comment (breed808) [inline collector/hyperv.go:318]: The metric unit will need to be added as a suffix for each of the metric names (E.G. `guest_run_time_percent`).

**Review by breed808 — CHANGES_REQUESTED** [review]
Thanks for submitting this! I've left one minor comment regarding the metric names, but everything else looks fine.

You'll want to rebase your branch on `master` in order to fix the broken CI job.

Apologies for the delay in reviewing this.

**Comment by gloyka** [comment]
Thank you very much!
I added "percent" suffix for metric names and rebase branch.

**Comment by breed808** [comment]
Thanks for that! The changes look good, but it appears that your rebase has resulted in a couple of commits from `master` being included.

Could you drop the two dependency commits? A `git rebase -i master` will do the trick.

**Review by breed808 — APPROVED** [review]

**Pull request merged**